components/sync15/src/sync.rs

```diff
@@ -69,6 +74,12 @@ pub fn synchronize(
         upload_info.successful_ids.len(),
         upload_info.failed_ids.len()
     );
+    // ideally we'd report this per-batch, but for now, let's just report it
```

Comment:
I think I had a way for this to work when I did the PostQueue but it's been a while and my head is too foggy for me to think through it now.

Reply:
Yeah, it requires a few changes to the sync API, we should file a issue. for properly reporting this